cluster.go

```diff
@@ -37,12 +38,21 @@ func NewClusterFromURL(url, databaseID, token string, timeout time.Duration) (*g
 }
 
 func NewCluster(dialer gocql.HostDialer, username, password string) *gocql.ClusterConfig {
-	cluster := gocql.NewCluster("127.0.0.1") // Placeholder, maybe figure how to make this better
+	// add multiple fake contact points to make gocql call the dialer multiple times (since the dialer will cycle through the contact points
+	cluster := gocql.NewCluster("127.0.0.1", "127.0.0.2", "127.0.0.3") // Placeholder, maybe figure how to make this better
```

Comment:
From a practical stand point I hope we could use `0.0.0.1`, `0.0.0.2`, ... because those aren't commonly used at it's easier to see something is off. There's also the "TEST-NET-1" range of IPs  `192.0.2.0` - `192.0.2.255`.

Reply:
Not a big deal though.

Reply:
@absurdfarce I know you advocated for `127.0.0.x` in the previous PR, any comments on this? I agree with Penick but I don't really have a strong opinion on this

Reply:
Apologies gents, I got buried under other things today and am just now getting around to this.  I'll try to take a look today and get some feedback here.

Reply:
I do think that there is some validity to what Penick said, if someone sees `0.0.0.1` in the gocql logs it will be easier to notice that something is wrong and it's definitely not intended for that address to be there while `127.0.0.1` could be interpreted as maybe some incorrect configuration or something like that.

> In the end I'd rather find some way to get Astra-aware HostInfo instance into gocql but that's a much more intrusive change... so this seems like a reasonable compromise.

That wouldn't be possible unless we forked gocql itself.